src/components/structures/RoomView.js

```diff
@@ -296,7 +360,7 @@ module.exports = React.createClass({
 
         if (ev.getSender() !== MatrixClientPeg.get().credentials.userId) {
             // update unread count when scrolled up
-            if (!this.state.searchResults && this.refs.messagePanel && this.refs.messagePanel.isAtBottom()) {
+            if (!this.state.searchResults && this.state.atBottom) {
```

Comment:
Aaah, this makes sense now! Can some of this information make its way into the code please? Perhaps renaming `atBottom` is all that will be needed to avoid confusion if it is just "primarily controls whether the 'scroll to bottom' knob appears"